src/managers/XCursorManager.cpp

```diff
@@ -511,8 +511,13 @@ std::vector<SP<SXCursors>> CXCursorManager::loadAllFromDir(std::string const& pa
 
     if (std::filesystem::exists(path) && std::filesystem::is_directory(path)) {
         for (const auto& entry : std::filesystem::directory_iterator(path)) {
-            if (!entry.is_regular_file() && !entry.is_symlink())
+            try {
```

Comment:
instead of try catch which kinda sucks can't you just `std::error_code e1, e2;` and pass them to the `is_*` fns?

then just `if (e1 || e2) continue`